exercises/space-age/pubspec.lock

```diff
@@ -0,0 +1,293 @@
+# Generated by pub
```

Comment:
This file can be removed as we recently started dropping them in favor of having the users generate lock files with more up to date dependencies.